.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,71 @@
+name: Build PR-Preview
+
+on:
+  pull_request_review:
+    types: submitted
+
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+concurrency:
```

Comment:
Could you please explain this?

Reply:
Actually I forgot about this, I also didn't know now why I used it, but during testing it was not creating any issues

Reply:
It ensures that only a specified number of workflow runs are executed at the same time, and it can help prevent redundant or conflicting runs, saving both time and resources.

Looks fine !